docs/contributing/guidelines.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Guidelines
+description: How to contribute to nf-co2footprint
+---
+
+# Contribution guidelines
+
+Hi! Thanks for contributing to the nf-co2footprint plugin for Nextflow 😊
+
+Please read the following to help us maintain and ensure high-quality code.
+
+## 💅 Follow commit style
+
+Commits should be structured like this: `<Type>: <Message>`, with the following types:
+
+- `Feature` representing an enhancement or new functionality
+- `Fix` when addressing a bug
+- `Documentation` when mainly improving comments, documentation files or docstrings
+- `Refactor` when moving stuff or changing naming, but essentially making no functional changes
+- `CI` when changing the continuous integration (guidelines, templates,...)
+- `Chore` when just doing something that has to be done once in a while or relates to other changes
+
+## 🖋️ Signing commits
+When submitting a pull-request (PR), please sign-off the [DCO](https://developercertificate.org/) to certify that you are the author of the contribution and you adhere to [Nextflow's open source license](https://github.com/nextflow-io/nextflow/blob/master/COPYING) by adding a `Signed-off-by` line to the contribution commit message. See [here](https://github.com/apps/dco) for more details.
+
+## 🚩 Github's issue tracking
+- When opening a pull-request that connects to an Issue, please mention it in the opening comment.
+- Assign labels / types to PRs / issues when possible
+
+## 🏗️ Create draft pull-requests
+- This tells us that you are working on something, even during early stage of development. Don't worry about everything being perfect already, that's what it's for.
+- If you are unable to finish the PR, someone may easily pick up the work where you left off.
+
+## 💬 Comment your code
+Use docstrings ...
```

Comment:
I would try to make the specific requirements here as short as possible

Reply:
and/or align to contribution guiledlines from Nextflow and nf-core as much as possible